### PR TITLE
Apps approval

### PR DESCRIPTION
**Pull request by whitneykidd**
Completes story 22, all tests passing

### CODE REVIEW

File: app/controllers/pets_controller.rb [hunk 1]
```diff
@@ -35,9 +35,27 @@ def destroy
...
+    if @pet.status == "Pending"
+      @pet.update(status: "Adoptable")
+    else 
+      @pet.update(status: "Pending")
+    end
+    # change_app_status
+    redirect_to "/pets/#{@pet.id}"
+  end
+
+  #  def change_app_status
```
Comment (whitneykidd) [inline app/controllers/pets_controller.rb:50]: Left in comments for reference during check-in

File: app/controllers/pets_controller.rb [hunk 2]
```diff
@@ -35,9 +35,27 @@ def destroy
...
+  #  self.pet_applications.update(approce)
+  # end
+
+
+
   private
 
   def pet_params
-    params.permit(:name, :image, :age, :description, :sex, :status, :favorite)
```
Comment (whitneykidd) [inline app/controllers/pets_controller.rb:41]: Removed favorite attribute from pets here

File: app/controllers/apps_controller.rb [hunk 3]
```diff
@@ -29,14 +30,13 @@ def create
   def show
     @app = Application.find(params[:id])
   end
-  # def create
```
Comment (janegreene) [inline app/controllers/apps_controller.rb:32]: replaced and improved above

File: app/views/pets/show.html.erb [hunk 4]
```diff
@@ -22,4 +28,5 @@
     <%= button_to "Remove From Favorites", "/favorites/#{@pet.id}?page=show", method: :delete %>
 <%= link_to 'Back to All Pets', "/pets" %>
 
-  <% end %></div>
+  <% end %></div></div>
```
Comment (whitneykidd) [inline app/views/pets/show.html.erb:31]: Cleaned up this view, thanks to @janegreene 👍

File: app/controllers/pets_controller.rb [hunk 5]
```diff
@@ -35,9 +35,27 @@ def destroy
...
 
+  def change_status
+    @pet = Pet.find(params[:id])
+
+    if @pet.status == "Pending"
+      @pet.update(status: "Adoptable")
+    else 
+      @pet.update(status: "Pending")
+    end
+    # change_app_status
```
Comment (janegreene) [inline app/controllers/pets_controller.rb:46]: Need to ask at check-in tomorrow

File: app/controllers/pets_controller.rb (hunk 2 above)
Comment (janegreene) [inline app/controllers/pets_controller.rb:41]: previously removed from table.

**Pull request merged**